src/js/components/molecules/cards/card-base/card-base.models.ts

```diff
@@ -0,0 +1,3 @@
+import { Card } from '@ultimaker/ultimaker.com-model-definitions/dist/molecules/cards/Card';
```

Comment:
Could you add `/** @format */` to new `.ts` files so they can be prettified using Prettier?

Doing it on the cli works like this:

```shell
$ npx prettier --write --insert-pragma --require-pragma=false path/to/file.ts
```

Reply:
done